boards.txt

```diff
@@ -10390,6 +10390,279 @@ datanoisetv_picoadk.menu.uploadmethod.picoprobe_cmsis_dap.upload.maximum_data_si
+datanoisetv_picoadk_v2.menu.flash.4194304_131072.build.eeprom_start=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_131072.build.fs_start=272490496
+datanoisetv_picoadk_v2.menu.flash.4194304_131072.build.fs_end=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_262144=4MB (Sketch: 3840KB, FS: 256KB)
+datanoisetv_picoadk_v2.menu.flash.4194304_262144.upload.maximum_size=3923968
+datanoisetv_picoadk_v2.menu.flash.4194304_262144.build.flash_total=4194304
+datanoisetv_picoadk_v2.menu.flash.4194304_262144.build.flash_length=3923968
+datanoisetv_picoadk_v2.menu.flash.4194304_262144.build.eeprom_start=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_262144.build.fs_start=272359424
+datanoisetv_picoadk_v2.menu.flash.4194304_262144.build.fs_end=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_524288=4MB (Sketch: 3584KB, FS: 512KB)
+datanoisetv_picoadk_v2.menu.flash.4194304_524288.upload.maximum_size=3661824
+datanoisetv_picoadk_v2.menu.flash.4194304_524288.build.flash_total=4194304
+datanoisetv_picoadk_v2.menu.flash.4194304_524288.build.flash_length=3661824
+datanoisetv_picoadk_v2.menu.flash.4194304_524288.build.eeprom_start=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_524288.build.fs_start=272097280
+datanoisetv_picoadk_v2.menu.flash.4194304_524288.build.fs_end=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576=4MB (Sketch: 3MB, FS: 1MB)
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576.upload.maximum_size=3137536
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576.build.flash_total=4194304
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576.build.flash_length=3137536
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576.build.eeprom_start=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576.build.fs_start=271572992
+datanoisetv_picoadk_v2.menu.flash.4194304_1048576.build.fs_end=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152=4MB (Sketch: 2MB, FS: 2MB)
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152.upload.maximum_size=2088960
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152.build.flash_total=4194304
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152.build.flash_length=2088960
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152.build.eeprom_start=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152.build.fs_start=270524416
+datanoisetv_picoadk_v2.menu.flash.4194304_2097152.build.fs_end=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728=4MB (Sketch: 1MB, FS: 3MB)
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728.upload.maximum_size=1040384
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728.build.flash_total=4194304
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728.build.flash_length=1040384
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728.build.eeprom_start=272621568
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728.build.fs_start=269475840
+datanoisetv_picoadk_v2.menu.flash.4194304_3145728.build.fs_end=272621568
+datanoisetv_picoadk_v2.menu.freq.150=150 MHz
```

Comment:
This still has the frequency menu items twice.  Please verify the Python logic you've implemented in the change.  Otherwise, much cleaner and ready to go!